lib/services/internalbroker.js

```diff
@@ -236,12 +236,18 @@ function clientConnect() {
 
   this.client.socket.on('error', function (e) {
     this.close();
-    this.log.warn('Error while trying to connect to ws://' + this.host + ':' + this.port, ': ', e.message);
-    this.log.warn('==> RECONNECTING IN ', this.client.retryInterval, 'ms');
+
+    this.kuzzle.pluginsManager.trigger('internalBroker:error',
```

Comment:
Nitpicking: this isn't actually an error, but a warning.
I downgraded this message level some time ago because workers and servers start order is random, and I didn't want people to fret over an "error" raised whenever a worker starts before its server.
As a side note for a later time: we should probably try to detect this case and make this message silent when this happens.